### PR TITLE
Feature/cart page

### PR DESCRIPTION
**Pull request by PYSchycho**

### CODE REVIEW

File: package-lock.json
Comment (ankur1778) [inline package-lock.json:1]: Delete package-lock.json

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,33 @@
...
+//     }
+//   }
+//   console.log("even:", even)
+//   console.log("odd:", odd)
+//   return (
+//     <div>
+//       Mehak
+//     </div>
+//   );
+// }
```
Comment (ankur1778) [inline src/App.js]: remove commented code

File: src/Pages/Address.jsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+      .required('City is required'),
+    state: Yup.string()
+      .matches(/^[A-Za-z\s]*$/, 'State only contain letters')
+      .required('State is required'),
+    PhoneNumber: Yup.string()
+      .matches(/^\d{10}$/, 'Phone Number must be 10 digits')
+      .required('Phone Number is required'),
+  });
+  const onSubmit = (values, { resetForm }) => {
+    console.log('Form data:', values);
```
Comment (ankur1778) [inline src/Pages/Address.jsx]: remove logs

File: src/Pages/Address.jsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    const cleanedValue = value.replace(/\D/g, '');
+    if (cleanedValue.length <= 10) {
+      setFieldValue(name, cleanedValue);
+    }
+  };
+  useEffect(() => {
+    const storedData = localStorage.getItem('address');
+    if (storedData) {
+      const parsedData = JSON.parse(storedData);
+      console.log('stored Address:', parsedData);
```
Comment (ankur1778) [inline src/Pages/Address.jsx]: remove logs

File: src/Pages/Login.jsx [hunk 4]
```diff
@@ -1,113 +1,96 @@
...
                 headers: {
                     'Content-Type': 'application/json',
                 },
                 body: JSON.stringify(payload),
             });
-            console.log(response, "res");
             if (!response.ok) {
-                const data = await response.json();
-                alert('Login Failed: ' + (data.message || 'Unknown error'));
+                console.log("Errors");
```
Comment (ankur1778) [inline src/Pages/Login.jsx]: remove logs

File: src/Pages/Login.jsx [hunk 5]
```diff
@@ -1,113 +1,96 @@
...
                 return;
             }
-
             const data = await response.json();
             console.log(data, 'Login Success');
```
Comment (ankur1778) [inline src/Pages/Login.jsx]: remove logs

File: src/Pages/Login.jsx [hunk 6]
```diff
@@ -1,113 +1,96 @@
...
-
             if (data.accessToken) {
                 localStorage.setItem('accessToken', data.accessToken);
                 localStorage.setItem('username', values.username);
                 localStorage.setItem('password', values.password);
                 navigate("/");
             } else {
-                alert('Login Failed: ' + (data.message));
+                console.log("Errors");
```
Comment (ankur1778) [inline src/Pages/Login.jsx:30]: remove logs

File: src/Pages/Main.jsx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+const Main = () => {
+  return (
+    <BrowserRouter
+      future={{
+        v7_startTransition: true,
+        v7_fetcherPersist: true,
+        v7_normalizeFormMethod: true,
+        v7_partialHydration: true,
+        v7_relativeSplatPath: true,
+        v7_skipActionErrorRevalidation: true,
```
Comment (ankur1778) [inline src/Pages/Main.jsx]: Please explain me this code.

File: src/Pages/Payment.jsx [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+import * as yup from 'yup';
+import { useNavigate } from 'react-router-dom';
+
+const Payment = () => {
+  const initialValues ={
+    paymentMethod:''
+  };
+  const navigate = useNavigate()
+  const onSubmit=(value) =>{
+    console.log("form :", value)
```
Comment (ankur1778) [inline src/Pages/Payment.jsx]: remove logs

**Review by ankur1778 — CHANGES_REQUESTED** [review]
Please check all the comments and resolve them

File: src/Pages/Main.jsx (hunk 7 above)
Comment (PYSchycho) [reply src/Pages/Main.jsx]: sir this code i will write because if i will get login than it will take me an error of react version update

**Review by ankur1778 — APPROVED** [review]

**Pull request merged**